src/main/java/com/conveyal/gtfs/validator/MTCValidator.java

```diff
@@ -39,6 +42,32 @@ public void validate() {
             validateFieldLength(trip, trip.trip_headsign, 120);
             validateFieldLength(trip, trip.trip_short_name, 50);
         }
+
+        // Validate that calendars apply to at least one day of the week.
+        for (Calendar calendar : feed.calendars) {
+            validateCalendarDays(calendar);
+        }
```

Comment:
I don't think this needs to be an MTC-specific validator. I think this should apply across the board. Sorry if that wasn't clear.